driver/pelican/main.go

```diff
@@ -85,6 +94,7 @@ func main() {
 		fmt.Println("Transforming", pelican.Name, "=>", name)
 		tstatIfaces[i] = service.RegisterInterface(name, "i.xbos.thermostat")
 		drstatIfaces[i] = service.RegisterInterface(name, "i.xbos.demand_response")
+		schedstatIfaces[i] = service.RegisterInterface(name, "i.xbos.schedule")
```

Comment:
We'll need to add a specification for this interface to the XBOS repository.

Reply:
https://github.com/SoftwareDefinedBuildings/XBOS/issues/78 @john-b-yang  can you document the interface at the XBOS repo?

Reply:
No problem, I will fork XBOS and make a pull request with the new .yaml files soon containing the configurations and formats.

Reply:
@gtfierro Just created a pull request on the XBOS repo with the new YAML files at SoftwareDefinedBuildings/XBOS#80!